app/src/androidTest/java/com/example/unlibrary/LibraryFlowTest.java

```diff
@@ -0,0 +1,542 @@
+package com.example.unlibrary;
+
+
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+
+import androidx.test.espresso.DataInteraction;
+import androidx.test.espresso.ViewInteraction;
+import androidx.test.filters.LargeTest;
+import androidx.test.runner.AndroidJUnit4;
```

Comment:
Change the import to use the same Junit runner as mock login, idk how this actually affects things but to be consistent. and it makes line 40 not crossed out 
```suggestion
import androidx.test.ext.junit.runners.AndroidJUnit4;
```